frontends/web/src/routes/account/add/add.tsx

```diff
@@ -58,46 +60,35 @@ class ManageAccount extends Component<Props, State> {
     public componentDidMount() {
         backendAPI.getSupportedCoins()
             // TEST with only 1 coin
-            // .then(() => (['btc']))
+            // .then(() => ([{coinCode: 'tbtc' as accountApi.CoinCode, name: 'Bitcoin Testnet', canAddAccount: true}]))
             .then((coins) => {
                 const onlyOneSupportedCoin = (coins.length === 1);
```

Comment:
nit: no parens () needed

Reply:
yes I know, but I always add them if it helps me read it.

The transpiler/bundler will remove/rewrite this anyway.

Reply:
removed